index.js

```diff
@@ -31,10 +52,11 @@ module.exports = {
         }
       };
 
-      return new TemplateLinter(tree, {
+      return TemplateLinter.create(tree, {
         annotation: 'TemplateLinter',
         templatercPath: this.project.root + '/.template-lintrc',
-        generateTestFile: this.project.generateTestFile,
+        testGenerator: this._options.testGenerator,
+        group: (this._options.group !== false) ? type : undefined,
```

Comment:
@rwjblue I've renamed the option in the Broccoli plugin to `groupName` to better reflect that they expect different types. I'm also excluding the `groupName` from the test suite name if it equals `templates` which seems to be the default in most Ember projects.